terraspace/app/stacks/shared/tfvars/base.tfvars

```diff
@@ -23,5 +23,6 @@ config_bucket_read_policy_name     = "<%= expansion(':ENV') %>-ep-s3-config-peer
 dotstorage_bucket_read_policy_name = "<%= expansion(':ENV') %>-ep-s3-dotstorage-buckets-read"
 storage_bucket_names               = [
   "dotstorage-<%= expansion(':ENV') %>-0",
-  "dotstorage-<%= expansion(':ENV') %>-1"
-]
+  "dotstorage-<%= expansion(':ENV') %>-1",
+  "carpark-<%= expansion(':ENV') %>-0"
```

Comment:
will need to add this to `prod.tfvars` in order to take effect

Reply:
@blucas that's not true, prod.tfvars isn't overriding `storage_bucket_names`

Reply:
This branch is out of date with `main`. `prod.tfvars` _is_ overriding `storage_bucket_names`